runtime/src/bpf_helper.cpp

```diff
@@ -77,11 +78,11 @@ uint64_t bpftime_get_prandom_u32()
 }
 
 uint64_t bpftime_ktime_get_coarse_ns(uint64_t, uint64_t, uint64_t, uint64_t,
```

Comment:
Do not change this function. This is an eBPF helper and designed to get coarse nanoseconds, the only way to get this value is to call clock_gettime